packages/react-embed/examples/event-log/src/index.tsx

```diff
@@ -17,6 +16,9 @@ interface LoggedEvent {
 
 const App = () => {
   const [loggedEvents, setLoggedEvents] = useState<LoggedEvent[]>([]);
+  useEffect(() => {
```

Comment:
Test useEffect double rendering.

Reply:
Maybe add inline comments for this? It doesn't tell me much when I read the code alone.